plugins/certifier/certifier.cc

```diff
@@ -211,7 +211,7 @@ class SslLRUList
 
         // Remove oldest node if size exceeds limit
         if (++size > limit) {
-          TSDebug(PLUGIN_NAME, "Removing %s", tail->commonName.c_str());
+          TSDebug(PLUGIN_NAME, "%s: removing '%s'", __func__, tail->commonName.c_str());
```

Comment:
Let's definitely remove explicit function names, but I don't think we need to add them to the log via `__func__` either since we support automatically adding them:
https://docs.trafficserver.apache.org/en/latest/admin-guide/files/records.yaml.en.html#proxy-config-diags-show-location

Or does that not apply to plugin messages?

Reply:
Yeh `TSDebug` doesn't know the source location. It wouldn't be too hard to fix that though.
```C
tsapi void TSDebug(const char *tag, const char *format_str, ...) TS_PRINTFLIKE(2, 3);
```

Reply:
Ah, OK. I'll leave it up to you then whether you want to keep the `__func__` here or add TSDebug support for it in a separate PR.

Thanks for looking into that.

Reply:
It would definitely be a separate PR.